dashboard/client/src/components/ActorTable.component.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import { Actor } from "../type/actor";
+import ActorTable from "./ActorTable";
+describe("ActorTable", () => {
+  const actors: { [actorId: string]: Actor } = {
```

Comment:
move this at the top of the file and it should be all caps `ACTORS` as a constant